frontend/src/metabase/admin/people/components/NudgeToPro.styled.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+import ExternalLink from "metabase/core/components/ExternalLink";
+
+export const NudgeCard = styled.div`
+  background-color: ${color("bg-light")};
+  border-radius: 6px;
+  padding: 20px 24px;
+  margin-top: 32px;
+  display: flex;
+  flex-direction: column;
+`;
+
+export const Description = styled.div`
+  margin-top: 16px;
+`;
+
+export const Subtitle = styled.div`
+  margin-top: 8px;
+  font-weight: 700;
+`;
+
+export const ProLink = styled(ExternalLink)`
```

Comment:
It's unfortunate that we can't use `<Button as={Link} />` here as that would save us from having to re-write these styles, but it seems it doesn't work with external links.

In the future we should be able to achieve this functionality with [Mantine's Button component](https://mantine.dev/core/button/#polymorphic-component). I think it's worth flagging to @kdoh when he integrates it into our codebase that it should work with external links for use cases like this.